lib/update-call-status.js

```diff
@@ -1,8 +1,8 @@
 const {makeCallKey, noopLogger, filterNullsAndObjects, CALL_SET} = require('./utils');
 const debug = require('debug')('jambonz:realtimedb-helpers');
 const assert = require('assert');
-const MAX_CALL_LIFETIME = 12 * 60 * 60;                 // calls live 12 hours max in redis
-const MAX_CALL_LIFETIME_AFTER_COMPLETED = 1 * 60 * 60;  // or 1 hour after completion
+const MAX_CALL_LIFETIME = process.env.JAMBONES_MAX_CALL_LIFETIME || 12 * 60 * 60;                         // calls live default 12 hours max in redis
+const MAX_CALL_LIFETIME_AFTER_COMPLETED = process.env.MAX_CALL_LIFETIME_AFTER_COMPLETED || 1 * 60 * 60;   // or default 1 hour after completion
```

Comment:
for consistency could you also prefix this env with JAMBONES?

Reply:
Typo sorry, should have had the prefix, will do.